### PR TITLE
Telemetry

### PR DESCRIPTION
**Pull request by greer184**
Some stuff that Ryan wanted to add to the PR.

### CODE REVIEW

File: services/comms-service/src/telemetry.rs [hunk 1]
```diff
@@ -24,18 +24,26 @@ use std::sync::{Arc, Mutex};
 pub struct CommsTelemetry {
     /// Errors that have occured within the communication service.
     pub errors: Vec<String>,
+    /// Number of packets uplinked where an error occured.
```
Comment (srjustice) [inline services/comms-service/src/telemetry.rs]: I think "Number of bad uplink packets" may sound a bit better.

File: services/comms-service/src/telemetry.rs [hunk 2]
```diff
@@ -24,18 +24,26 @@ use std::sync::{Arc, Mutex};
...
+    pub failed_packets_up: i32,
+    /// Number of packets downlinked where an error occured.
```
Comment (srjustice) [inline services/comms-service/src/telemetry.rs]: I think "Number of bad downlink packets" may sound a bit better.

File: services/comms-service/src/telemetry.rs (hunk 1 above)
Comment (greer184) [reply services/comms-service/src/telemetry.rs]: Yes

File: services/comms-service/src/telemetry.rs (hunk 2 above)
Comment (greer184) [reply services/comms-service/src/telemetry.rs]: also yes

**Review by srjustice — APPROVED** [review]

**Pull request merged**